### DISCUSSION TITLE
Auto corrections not displayed (Carelink Follower)

### DISCUSSION THREAD

**Discussion by Harrinator** [NightscoutFoundation/xDrip#3791] [Bug reports] [answered]
Since November 26th 2024 (evening; Germany) xDrip+ has stopped displaying Auto-Corrections.  

Could you please fix the problem or tell me what I can do to fix it?

**Xdrip+ version:** see screenshot
**Insulin pump:** Medtronic 780g
**Sensor:** Guardian 4
**Hardware data source:**  CareLink Follower
**Phone:** google Pixel 6a
**Android version:** 14

![Screenshot_20241130-140821](https://github.com/user-attachments/assets/930fe264-5196-4c9f-af44-ca0ad9052aa5)

**Comment by Navid200** [comment, marked as answer]
Can you update at least to the November 17, 2024 release?  
https://navid200.github.io/xDrip/docs/ReleaseNotes.html

**Reply by Harrinator** [reply]
Thank you for your quick reply. Will update the app and will report here later.

**Reply by Harrinator** [reply]
It worked! Thank you very much, Navid.

**Reply by Navid200** [reply]
Great!
You're welcome
